ext/standard/tests/strings/crypt_blowfish.phpt

```diff
@@ -2,7 +2,7 @@
 Official blowfish tests (http://cvsweb.openwall.com/cgi/cvsweb.cgi/Owl/packages/glibc/crypt_blowfish/wrapper.c)
 --SKIPIF--
 <?php
-if (!function_exists('crypt') || !defined("CRYPT_BLOWFISH")) {
+if (!defined("CRYPT_BLOWFISH")) {
```

Comment:
This check is also no longer necessary. Nowadays we guarantee availability of all hashes.

Reply:
[This is how](https://github.com/php/php-src/blob/master/ext/standard/crypt.c#L75) we register our constants? I can prepare a PR remove `defined` checks as well :smile: